modules/heat_conduction/test/tests/NAFEMS/transient/T3/nafems_t3_edge2_coarse.i

```diff
@@ -0,0 +1,102 @@
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 5
+  xmin = 0.0
+  xmax = 0.1
+  elem_type = EDGE2
+[]
+
+[Variables]
+  [./temp]
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = 0.0
+  [../]
+[]
+
+[Functions]
+  [./temp_func]
+    type = ParsedFunction
+    value = '100.0 * sin(pi*t/40)'
+  [../]
+[]
+
+[BCs]
+  [./FixedTempLeft]
+    type = DirichletBC
+    variable = temp
+    boundary = left
+    value = 0.0
+  [../]
+  [./FunctionTempRight]
+    type = FunctionDirichletBC
+    variable = temp
+    boundary = right
+    function = temp_func
+  [../]
+[]
+
+[Kernels]
+  [./heat]
+    type = HeatConduction
+    variable = temp
+  [../]
+  [./HeatTdot]
+    type = HeatConductionTimeDerivative
+    variable = temp
+  [../]
+[]
+
+[Materials]
+  [./heat]
+    type = HeatConductionMaterial
+    block = 0
+    specific_heat = 440.5
```

Comment:
Why not just stick thermal conductivity, specific heat, and density directly into the kernels since they are constant?